homeassistant/components/switch/rachio.py

```diff
@@ -19,7 +19,7 @@
 CONF_MANUAL_RUN_MINS = 'manual_run_mins'
 DEFAULT_MANUAL_RUN_MINS = 10
 
-MIN_UPDATE_INTERVAL = timedelta(minutes=5)
+MIN_UPDATE_INTERVAL = timedelta(seconds=5)
```

Comment:
Rachio offers an API for free but hitting it every 5 seconds is going to get us banned.